lib/pdf.js

```diff
@@ -35,6 +35,8 @@ function PDF (html, options) {
   if (this.options.filename) this.options.filename = path.resolve(this.options.filename)
   if (!this.options.phantomPath) this.options.phantomPath = phantomjs && phantomjs.path
   this.options.phantomArgs = this.options.phantomArgs || []
+
+  if (this.options.localUrlAccess) this.options.phantomArgs.push('--local-url-access=false')
```

Comment:
Am I crazy or is this inverted? if localUrlAccess is false (which it is by default, for security reasons), this option never gets set for phantomjs to disallow local url access? Are the tests inverted as well?

Reply:
hah. oops

Reply:
In the case you didn't already go ahead and fix it, I created a PR here to address this https://github.com/marcbachmann/node-html-pdf/pull/623